lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback_file.inc

```diff
@@ -58,17 +58,16 @@ class ReportbackFileEntity extends Entity {
       // Store that instead.
       $this->review_source = $values['source'];
     }
-    if ($this->status === 'flagged') {
```

Comment:
This conditional was added as a sanity check: If an app posts a Review with a non-`flagged` status but still sends a `flagged_reason` -- it's assuming it actually shouldn't be flagged.  Any reason to take it out?